src/components/StatisticCard/_statistic-card.scss

```diff
@@ -0,0 +1,58 @@
+.cc-statistic-card {
+  @include mq($until: sm) {
+    margin: $grid-gutter-sm / 4;
+    margin-bottom: calc(3 * var(--space-lg));
+    margin-top: $grid-gutter-sm;
+  }
+
+  @include mq(sm) {
```

Comment:
This component isn't arranged into columns until 1024+ ('md' breakpoint)

https://app.zeplin.io/project/5b4e06b48ae4580d4871178a/dashboard?q=statistic%20cards